src/app/_components/_sharedcomponents/Cards/LeaderCard/LeaderCard.tsx

```diff
@@ -0,0 +1,150 @@
+import React from "react";
+import { Card, CardContent, Typography, Box } from "@mui/material";
+import { ICardData } from "../CardTypes";
+import { useGame } from "@/app/_contexts/Game.context";
+import { s3CardImageURL } from "@/app/_utils/s3Utils";
+
+interface ILeaderCardProps {
+    isLobbyView?: boolean;
+    title?: string;
+    card: ICardData;
+}
+
+const LeaderCard: React.FC<ILeaderCardProps> = ({ isLobbyView = false, title, card }) => {
```

Comment:
I don't think these cards need to be split into two different components. They're quite similar to each other in most regards. Same as the GameCard what I'd rather see is that we keep the card as a LeaderBaseCard and have the props be the elements of the card that we want shown. In this case it would be The player nameplate and the damage counter. Otherwise these cards display the same way.